[docs] Describe how to mock part of response headers (#6089)
DevExpress/testcafe: docs/articles/templates/notes-warnings/requesthook-onresponse-no-stub-headers.md

> > Important! You can not change response body or its headers inside a `RequestHook.onResponse()` call.

Reviewer: You can change headers inside of request hooks.

`> Important! You can not change response body or its status code inside a `RequestHook.onResponse()` call.`

Author: ```suggestion
> Important! You cannot change the response body or its status code inside a `RequestHook.onResponse()` call.
```

> > Important! You can not change response body or its headers inside a `RequestHook.onResponse()` call.

Reviewer 2: ```suggestion
> Important! You cannot change the response body or its headers inside a `RequestHook.onResponse()` call.
```

> > Important! You can not change response body or its headers inside a `RequestHook.onResponse()` call. If you edit properties of `event.headers` or `event.body` inside `onResponse()`, TestCafe ignores these changes.

Reviewer: The `RequestHook.onResponse` method takes a response from server. If you edit properties of `event.headers` or `event.body` inside `onResponse()` it affects the current request hook only. This modification will not affect the tested application response. So, the data in `onResponse` method of other request hooks is not changed.

>

> > Learn how to change or delete response headers in [Change or Delete Response Headers]({{ include.href }}).

Reviewer 2: ```suggestion
> Refer to the following topic for instructions on how to change or delete response headers: [Change or Delete Response Headers]({{ include.href }}).
```

> > Learn how to stub response headers in [Stub Response Headers]({{ include.href }}).

Reviewer: See comments above
